dspace/config/modules/authentication-shibboleth.cfg

```diff
@@ -130,6 +130,10 @@ lastname-header = SHIB-SURNAME
 # Additional user attributes mapping, multiple attributes may be stored
 # for each user. The left side is the Shibboleth-based metadata Header
 # and the right side is the eperson metadata field to map the attribute to.
+# Attention: This will not work out of the box for arbitrary new attributes.
+# Be sure, that your attribute is listed in getMDValueByLegacyField()
+# in file DSpaceObject.java.
+# https://github.com/DSpace/DSpace/blob/dspace-5.6/dspace-api/src/main/java/org/dspace/content/DSpaceObject.java#L1354
```

Comment:
Right, I agree, that this would be better and more elaborated, but this also would need time and some effort...